src/main/java/InvalidCommandDukeException.java

```diff
@@ -0,0 +1,9 @@
+/**
+ * Represents an exception that is thrown when an invalid command is entered
```

Comment:
I like it that you define subclasses of DukeException. This makes the logic of exception handling much clearer!